__tests__/main.test.ts

```diff
@@ -54,6 +54,11 @@ describe('run', () => {
 
   describe('when the run follows the happy path', () => {
     beforeEach(() => {
+      jest.spyOn(ApiClient.prototype, 'getJobDetails').mockImplementationOnce(
+        jest.fn(async () => {
+          return {'package-manager': 'npm_and_yarn'} as JobDetails
+        })
+      )
```

Comment:
At first I tried pasting this `spyOn` in the `beforeEach` at the top of the file, but then the error tests wouldn't let me re-spy/mock them again. 🤷